AGENTS.md

```
@@ -28,3 +28,104 @@
+# Option B: Amend the last commit
+# Use when only the last commit needs fixing
+git commit --amend -m "feat: corrected commit message"
+
+# Option C: Use fixup commits (advanced)
+# Create fixup commits, then auto-squash them
+git commit --fixup=HEAD~2  # Marks as fixup of earlier commit
+git rebase --autosquash main  # Auto-squashes fixups (non-interactive)
+```
+
+**For human reviewers only:**
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)